docs/installation/installing.rst

```diff
@@ -27,6 +27,17 @@ Supported Versions and Interoperability
 
 .. _install_ksql-server:
 
+-------------------------
+Installation Instructions
+-------------------------
+
+Follow the instructions at :ref:`installing_cp`.
+
+It is also possible to install KSQL individually through :ref:`standalone packages <available_packages>`.
```

Comment:
i added an anchor to the installation specifically for KSQL.
```
You can install standalone KSQL by using the :ref:`component packages <confluent-ksql-package>`.
```

Reply:
@joel-hamill : This anchor is not in the `4.1.0-post` branch of `confluentinc/docs`.  I know because I checked prior to this PR. Seems like the anchor was removed again after https://github.com/confluentinc/docs/pull/888 was merged?

Reply:
ping @joel-hamill

Reply:
@miguno i see it there https://github.com/confluentinc/docs/blame/4.1.0-post/installation/available_packages.rst#L185